README.md

```diff
@@ -264,9 +264,9 @@ extension MyViewController: ShopifyCheckoutSheetKitDelegate {
 
 App developers can use [lifecycle events](#monitoring-the-lifecycle-of-a-checkout-session) to monitor and log the status of a checkout session. 
 
-**To safeguard user privacy, Web Pixel events will not be dispatched from within the Checkout webview.** Instead, these events will be relayed back to your application through the checkoutDidEmitWebPixelEvent delegate hook. The responsibility then falls on the application developer to ensure adherence to Apple privacy protocols before disseminating these events to third-party providers.
+**For behavioural monitoring, Checkout [Web Pixel events](https://shopify.dev/docs/api/web-pixels-api/standard-events) will be relayed back to your application through the `checkoutDidEmitWebPixelEvent` delegate hook. The responsibility then falls on the application developer to ensure adherence to Apple's privacy policy and local regulations like GDPR and ePrivacy directive before disseminating these events to first-party and third-party systems.
```

Comment:
We link to standard events, but custom events will also be relayed. In case we think that's worth noting?

Reply:
Equivalent android PR here - https://github.com/Shopify/checkout-sheet-kit-android/pull/58